docs/FAQ.md

```diff
@@ -140,7 +140,7 @@ If all maintainer feedback has been addressed and all tests are passing, bump it
 
 ## Can I edit formulae myself?
 
-Yes! It’s easy! Just `brew edit <formula>`. You don’t have to submit modifications back to `homebrew/core`, just edit the formula to what you personally need and `brew install <formula>`. As a bonus, `brew update` will merge your changes with upstream so you can still keep the formula up-to-date **with** your personal modifications!
+Yes! It’s easy! Just `brew edit <formula>`. You don’t have to submit modifications back to `homebrew/core`, just edit the formula to what you personally need and `HOMEBREW_NO_INSTALL_FROM_API=1 brew install <formula>`. As a bonus, `brew update` will merge your changes with upstream so you can still keep the formula up-to-date **with** your personal modifications!
```

Comment:
Stupid question: does `brew update` need `HOMEBREW_NO_INSTALL_FROM_API=1` here too?

Reply:
Nope, `HOMEBREW_NO_INSTALL_FROM_API` needs to be set only for the `install`. I don't think it does anything to `edit`.

Reply:
This piece of documentation is out-of-date so I'm not sure I would trust it in this case. The `brew update` command doesn't update the `homebrew/core` and `homebrew/cask` taps when using the API so your changes shouldn't get overwritten but no upstream changes will get merged in either.

https://github.com/Homebrew/brew/blob/1cf53b4281e7bc22a8d7fc915fc6ba4576f8ef88/Library/Homebrew/cmd/update-report.rb#L159

Reply:
To be clear I'm not sure that changing that is within the scope of this PR but it does make me wonder how we should handle these things going forward. Passing `HOMEBREW_NO_INSTALL_FROM_API` implicitly every time the `brew edit` command is called seems to make sense until you realize that it causes unexpected behavior when using other commands.

Reply:
@apainintheneck @EricFromCanada any suggestions on how to update this documentation to be both correct and helpful would be handy.

> Passing `HOMEBREW_NO_INSTALL_FROM_API` implicitly every time the `brew edit` command is called seems to make sense until you realize that it causes unexpected behavior when using other commands.

Agreed this could be tweaked/improved. Don't think it's worth reverting this, though.